relayer/relays/beefy/on-demand-sync.go

```diff
@@ -86,98 +86,72 @@ func (relay *OnDemandRelay) Start(ctx context.Context) error {
 		return fmt.Errorf("create gateway client: %w", err)
 	}
 	relay.gatewayContract = gatewayContract
+	// Check the on-demand sync every 30 minutes
+	ticker := time.NewTicker(time.Minute * 30)
```

Comment:
The relayer already blocks for around 30 minutes when calling submitInit/Final.

Adding another 30 minute delay means that in the worst case a user needs to wait 60 minutes (30 + 30).

If you reduce this ticker time to 60s then it's perfect 👌

Reply:
https://github.com/Snowfork/snowbridge/pull/1530/commits/8a3dd6743dbb33f1655cb52bffa7209a1844de12